Dockerfile

```diff
@@ -6,5 +6,6 @@ COPY environment.yml /
 COPY environment-percolator.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 RUN conda env create -f /environment-percolator.yml && conda clean -a
-ENV PATH /opt/conda/envs/nf-core-mhcquant-1.0.0/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-mhcquant-1.1.0/bin:$PATH
 ENV PATH /opt/conda/envs/nf-core-mhcquant-percolator-1.0dev/bin:$PATH
+RUN mhcflurry-downloads fetch models_class1
```

Comment:
hmm.. not sure if thats possible, ill check

Reply:
@lkuchenb

Reply:
Not sure if this would be such a good idea in the case of MHCflurry. There is no way to guarantee the package content consistency once it is installed and the user starts using the download tool.

Reply:
So this script is part of mhcflurry? Then we should package this, and run the download in the process requireing the models... 
(and save some info which model was used...)

Reply:
The script is part of mhcflurry and also provided by the mhcflurry bioconda package.

Reply:
ok right, that should be doable instead.